Assets/Scripts/Framework/GeoLocation/RouteHighlighter.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Collections.Generic;
+using System.Collections;
+using UnityEngine;
+
+namespace Framework.GeoLocation
+{
+    public sealed class RouteHighlighter : MonoBehaviour
+    {   
+        [SerializeField] private List<Transform> routePoints;
+        [SerializeField] private LineRenderer lineRenderer;
```

Comment:
```suggestion
        [SerializeField] private LineRenderer route;
```
[code convention](https://github.com/Team-Swamp/Codename-BDLMTW/wiki/Code-convention#hungarian-notation)